.eslintrc.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
95% of this file was autogenerated when running `eslint --init` (it asks questions for what types of code you have), and most of the "best" lint settings come with the `"eslint:recommended"` extension.